proto/desmos/profiles/v1beta1/query_relationships.proto

```diff
@@ -18,17 +18,28 @@ message QueryUserRelationshipsRequest {
 
   // address of the user to query the relationships for
   string user = 1;
+
+   // subspace to query the relationships for
+  string subspace = 2;
+
+  // pagination defines an optional pagination for the request.
+  cosmos.base.query.v1beta1.PageRequest pagination = 3;
 }
 
 // QueryUserRelationshipsResponse is the response type for the
 // Query/UserRelationships RPC method.
 message QueryUserRelationshipsResponse {
   string user = 1;
 
+  string subspace = 2;
```

Comment:
I think we can remove this since the subspace is present inside each relationship object anyway